book/routing.rst

```diff
@@ -223,7 +146,7 @@ A basic route consists of just two parts: the ``path`` to match and a
         class MainController extends Controller
         {
             /**
-             * @Route("/")
+             * @Route("/", name="_welcome")
```

Comment:
Let's not include this yet - I realize that it is defined in the other formats, but that's because we *have* to do that - we're not trying to teach routing naming here.